scripts/bfb-install

```diff
@@ -624,6 +626,8 @@ num_bfb=0
 num_rshim=0
 max_bfb=1
 max_rshim=1
+lfwp=0
+activate=
```

Comment:
Shouldn't activate=1 by default as mentioned above?

Reply:
Updated, removed the "default=1" in command help message.

Reply:
We need to differentiate the following two cases.  Set default value 0 will make it hard.
--lfwp   ===> indicate activate=1
--lfwp activate=0